src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -295,3 +295,204 @@ and if they have a value depending on the event origin.
 | Y
 | Y
 |=====
+
+[[microsoft_teams_configuration]]
+==== Microsoft Teams notification method
+
+ProActive provides a ready to use Third-Party script that allows users to send notifications to multiple Teams channels.
+
+The script interacts with Microsoft Graph API to:
+
+ - Connect as a user
+ - Send a message to multiple teams channels
+
+Some configuration must be done in ProActive and Microsoft beforehand but it consists in a nutshell of:
+
+- Configuring the notification service to execute the correct script with the correct setup
+- Create or retrieve app information from the Microsoft admin center
+- Provide these information to the script
+- Update User Settings in the notification portal
+- Add channel URLs to subscriptions
+
+We will go into details on how to achieve this configuration now.
+
+===== Configuring the notification service
+
+All the following configuration takes place in the Notification Service's properties file located
+----
+PROACTIVE_INSTALL_FOLDER/dist/war/notification-service/WEB-INF/classes/application.properties
+----
+
+All the relevant properties are located at the end of the file. We will detail them now.
+
+----
+notification.script.path=dist/war/notification-service/WEB-INF/classes/microsoftTeamsMessage.groovy
+# URL of the script, it will be processed if notification.script.path is empty
+notification.script.url=
+#optional parameter for specifying Script Engine
+notification.script.engine=groovy
+#if false execute the script one time per event even if there are several subscribers
+#if true execute the script for each subscribers for each event
+notification.script.multiple.execution=true
+----
+
+- *notification.script.path*: The notification service can execute a single Third-Party script, for all users. This property specifies the script that will be executed.
+    It can't be dynamically updated and requires a server restart to be changed.
+- *notification.script.url:* Allows specifying the script url instead of using a path. Not used here
+- *notification.script.engine:* Specifies the engine used to execute the script. Here it must be set to groovy as we are executing a groovy script.
+- *notification.script.multiple.execution:* Sets the execution mode to single or multiple execution. Explained <<execution-mode,here>>
+    We will be using this script in multiple execution mode as we want each user to be able to send notifications using their microsoft accounts on different channels
+
+Once the properties are updated as the code snippet above, you can save the file.
+
+===== Configuring Microsoft 365
+Because the script queries the Microsoft Graph API (a REST API) to interact with microsoft Teams, it needs to connect to Microsoft as a user (retrieve an authentication token) through a registered app.
```

Comment:
App or application ?

Reply:
if it's app (and in any case actually), it would be useful to add a link to the microsoft documentation which defines what an App or Application is

Reply:
I will add this link earlier in the doc https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app?tabs=certificate

Inside it says what it does to register and App, which is is the closest to a definition I can find